src/test/java/seedu/address/logic/parser/lesson/MarkLessonParticipationCommandParserTest.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.logic.parser.lesson;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.logic.parser.ParserUtil.MESSAGE_INVALID_INDEX;
+import static seedu.address.logic.parser.ParserUtil.MESSAGE_INVALID_POINTS;
+import static seedu.address.logic.parser.lesson.MarkLessonParticipationCommandParser.MESSAGE_TOO_MANY_PARTICIPATION_ARGUMENTS;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.lesson.MarkLessonParticipationCommand;
+import seedu.address.model.student.Name;
+
+public class MarkLessonParticipationCommandParserTest {
+    private static final Index VALID_INDEX = Index.fromOneBased(1);
+    private static final List<Name> VALID_NAMES = List.of(new Name("Alice Tan"), new Name("Benson Son"));
```

Comment:
Can consider using variables for student names

Reply:
Good idea. Done.